stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt

```diff
@@ -1,44 +1,79 @@
 package io.getstream.chat.android.compose.ui.components.messages
 
-import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
-import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
 import coil.compose.rememberImagePainter
 import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.compose.state.messages.MessageAlignment
 import io.getstream.chat.android.compose.ui.components.avatar.Avatar
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
 
+private const val DEFAULT_PARTICIPANTS_LIMIT = 4
+
 /**
- * Shows a row of participants in the message thread, if they exist.
+ * Represents the last two participants in the thread.
```

Comment:
4 by default?

Reply:
Good point - 2 just didn't make sense to me, so I put in 4 🤷 I don't have much preference on the default, so if you have any idea what it could be we can define it as such.

I'll update the docs however!